src/deployments.ts

```diff
@@ -2,6 +2,29 @@ import { get } from "./utils/fetch";
 import { endpointMap } from "./common";
 import { DeploymentList } from "./types/deployment";
 
-export const getDeploymentsList = () => {
-  return get<DeploymentList>(endpointMap.listDeployment);
+type listDeploymentParams = {
```

Comment:
Types should use PascalCase so can you make this `ListDeploymentParams`? Also, it should be `ListDeploymentsParams` ... plural because this is a list all deployments endpoint?